addons/im_livechat/static/src/scss/im_livechat_history.scss

```
@@ -35,3 +35,10 @@
         }
     }
 }
+
+aside {
+    &.o_kanban_aside_small {
+        --KanbanRecord__image-width: 36px;
+        --KanbanRecord__image-height: 36px;
+    }
+}
```

do not do that (I already saw that in another PR). These rules aren't scoped, so if someone, in another unrelated addon uses the (quite generic) classname o_kanban_aside_small, those rules will apply).
```suggestion
.whatever_classname_you_set_on_this_particular_kanban_view {
    .o_kanban_record > aside {
        --KanbanRecord__image-width: 36px;
        --KanbanRecord__image-height: 36px;
    }
}
```

---

I found another way to achieve this by passing `options="{'size': [36, 36]}"` in the field, like this:
`<field t-if="record.author_avatar.raw_value" name="author_avatar" widget="image" options="{'size': [36, 36]}" alt="Avatar" class="rounded"/>`
Is this a better approach, or should we stick with overriding the CSS only, considering that it would require creating a separate class and additional steps?

---

if it works, great :-)